lib/Core/injectTerms.ts

```diff
@@ -71,14 +77,28 @@ const injectTerms = (string: string, termDictionary: Term[]): string => {
   const injectedBoldSet = new Set();
   while (1) {
     let tooltipTerms = new Map<string, Term>();
+
     termDictionary.forEach((item: any) =>
-      tooltipTerms.set(item.term.toLowerCase(), item)
+      tooltipTerms.set(
+        i18next.exists(item.term)
+          ? i18next.t(item.term).toLowerCase()
+          : item.term.toLowerCase(),
+        item
+      )
     );
     // some help content things will have aliases / variants
+
     termDictionary.forEach(term => {
-      term.aliases?.forEach(alias => {
-        tooltipTerms.set(alias.toLowerCase(), term);
-      });
+      const aliasesTranslated = term.aliases
+        ? i18next.exists(term?.aliases)
+          ? i18next.t(term?.aliases, { returnObjects: true })
+          : term?.aliases
+        : [];
```

Comment:
When the `helpContentTerms` reference value in override file it will throw error if they doesn't exist in that file. In that case the value is not an array but a string
![override error](https://user-images.githubusercontent.com/19208948/89412317-243e9900-d727-11ea-89ad-aa57405fb44a.png)

```suggestion
      const aliasesTranslated = term.aliases
        ? i18next.exists(term.aliases)
          ? <Array<string>>i18next.t(term?.aliases, { returnObjects: true })
          : Array.isArray(term.aliases.slice())
          ? term.aliases
          : [term.aliases]
        : [];
```

Reply:
Thanks good pickup, I ran into this as well when testing out overrides but hadn't gone to fix it